src/generators/schema/article.php

```diff
@@ -55,11 +55,15 @@ public function generate() {
 			],
 			'headline'         => $this->helpers->schema->html->smart_strip_tags( $this->helpers->post->get_post_title_with_fallback( $this->context->id ) ),
 			'datePublished'    => $this->helpers->date->format( $this->context->post->post_date_gmt ),
-			'dateModified'     => $this->helpers->date->format( $this->context->post->post_modified_gmt ),
-			'mainEntityOfPage' => [ '@id' => $this->context->main_schema_id ],
-			'wordCount'        => $this->word_count( $this->context->post->post_content, $this->context->post->post_title ),
 		];
 
+		if ( \strtotime( $this->context->post->post_modified_gmt ) > \strtotime( $this->context->post->post_date_gmt ) ) {
+			$data['dateModified'] = $this->helpers->date->format( $this->context->post->post_modified_gmt );
+		}
+
+		$data['mainEntityOfPage'] = [ '@id' => $this->context->main_schema_id ];
+		$data['wordCount']        = $this->word_count( $this->context->post->post_content, $this->context->post->post_title );
```

Comment:
Not sure I get why these were moved out of the array above?

Reply:
So that we keep the same order of the node's attributes.

Because it could be not as readable if we had `dateModified` far from `datePublished`